CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
```

Comment:
I totally agree with the approach but I am not sure what was the criteria for getting some things in and leaving something out. age/disability/sex/gender/race/religion/identity/orientation are obvious, but why 'body size'  ? I mean, I am bald, should we add "hair status" , height, shoe size, etc.. ? Would it be more accurate to say "polite and respectful regardless of physical characteristics and appearance" ?

Reply:
This list is not meant to be exhaustive, but it has worked well for thousands of projects so far. I copied it directly from https://github.com/ContributorCovenant/contributor_covenant.

I don't think that shoe size is often used as a discriminator in open source communities. "body size" is there because fat-shaming is a legitimate problem (for more, see [this google search](https://www.google.com/search?client=firefox-b-1-d&q=fat+shaming)). Anorexia is also an issue, on the other side. I think it's a legitimate item.